Marsiple/App/Todo Form/TodoFormViewController.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  TodoFormViewController.swift
+//  Marsiple
+//
+//  Created by Matej Korman on 31/07/2018.
+//  Copyright © 2018 Matej Korman. All rights reserved.
+//
+
+import SnapKit
+
+class TodoFormViewController: UIViewController {
+    // NOTE: id is set to -1 so we can detect if we're creating new or changing existing todo (to detect wich initializer is used)
+    private var todo = Todo(id: -1, title: "", completed: false, userId: 1)
```

Comment:
I think that a more straightforward and understandable approach would be to initialise a new `Todo` instance via the `initialiser` that accepts an `optional` `Todo`, and keeping a `let isEditing: Bool` stored property (also initialised via the initialiser) that indicates whether or not the form is in `editing mode`.